freeipa/src/main/java/com/sequenceiq/freeipa/altus/FreeIpaUMSCleanupJob.java

```diff
@@ -66,18 +65,24 @@ protected void executeTracedJob(JobExecutionContext context) throws JobExecution
             for (UserManagementProto.MachineUser machineUser : machineUsers) {
                 String name = machineUser.getMachineUserName();
                 if (name.startsWith("freeipa-fluent") && !machineUserValues.contains(name)) {
-                    LOGGER.debug("Cannot found stack for machine user {} (account: {}). Checks that if it can be deleted.",
-                            name, accountId);
-                    Instant beforeTime = Instant.now().minus(umsCleanupConfig.getMaxAgeDays(), ChronoUnit.DAYS);
-                    Instant creationTime = new Date(machineUser.getCreationDateMs()).toInstant();
-                    if (creationTime.isBefore(beforeTime)) {
-                        altusMachineUserService.cleanupMachineUser(name, accountId);
-                    } else {
-                        LOGGER.debug("Machine user with name {} is not old enough yet, skipping cleanup. (account id: {})",
-                                name, accountId);
-                    }
+                    clearMachineUser(accountId, machineUser, name);
+                } else if (name.startsWith("freeipa-monitoring") && !machineUserValues.contains(name)) {
```

Comment:
why is this else-if necessary? it could be merged with the previous with on or condition